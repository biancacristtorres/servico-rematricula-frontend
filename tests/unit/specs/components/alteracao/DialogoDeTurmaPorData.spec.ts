import {
  DialogoDeTurmaPorDataFactory,
} from '../../../factory/components/DialogoDeTurmaPorDataFactory';

describe('Componente de diÃ¡logo de turma por data', () => {
  const factory = new DialogoDeTurmaPorDataFactory();

  it('Snapshot', async (done) => {
    factory.dadoQueExistemHorariosDeTurmaPorData();

    await factory.montarComponente();

    expect(factory.componente).toMatchSnapshot();
    done();
  });

});
