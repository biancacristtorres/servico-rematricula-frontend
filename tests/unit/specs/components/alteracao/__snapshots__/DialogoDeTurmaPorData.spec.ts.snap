// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de diálogo de turma por data Snapshot 1`] = `
<div id="dialogo-de-turma-por-data">
  <p class="body-2 mb-0">
    Cálculo
  </p>
  <p class="body-2 pt-0">
    Turma: Turma
  </p>
  <div>
    <p class="body-2">
      Segunda-feira:
    </p>
    <ul class="pl-0 mb-3">
      <li class="body-1">
        Segunda 13 horas
      </li>
    </ul>
  </div>
</div>
`;
